index.php

```diff
@@ -9,10 +9,60 @@
     <div class="wrapper">
         <div id="logo-bar">
             <?php include 'logo-include.php'; ?>
-            <h1> Flock Information Database Home </h1>
-        </div>        
+            <h1> Home </h1>
+        </div>
+
         <div class="content">
-            <h2> Stick a login form here </h2>
+            <table id="register-login">
+            <tr>
+            <td><form method="post" action="index.php">
+                <table id="inner-register">
+                    <th colspan="2"><h3>New users register here.</h3></th>
+                    <tr>
+                        <td><b>Username</b></td>
+                        <td><input type="text" name="username">
+                    </tr>
+                    <tr>
+                        <td><b>Password</b></td>
+                        <td><input name="password" type="password"></input></td>
+                    </tr> <br/>
+                    <tr>
+                        <td><b>User Type</b></td>
+                        <td><input type="text" name="user_type">
+                    </tr>
+                    <tr>
+                        <td><b>Email</b></td>
+                        <td><input type="text" name="email">
+                    </tr>
+                    <tr>
+                        <td><td><input type="submit" value="Register"/>
+                    </tr>
+                </table>
+            </form></td>
+
+            <td><form method="post" action="redirectPostLogin.php" >
+                <table id="inner-login">
+                    <th colspan="2"><h3>Existing users login here.</h3></th>
+                    <tr>
+                        <td><b>Username</b></td>
+                        <td><input type="text" name="username">
+                    </tr>
+                    <tr>
+                        <td><b>Password</b></td>
+                        <td><input name="password" type="password"></input></td>
+                    </tr> <br/>
```

Comment:
We should try to avoid `<br />` tags, unless it's part of a paragraph. Adding proper spacing with `margin` or `padding` in the css is probably the way to go. With `<br />` tags its hard to control how much space you want.